core/objectmodelbase.h

```diff
@@ -59,6 +60,8 @@ class ObjectModelBase : public Base
      */
     explicit ObjectModelBase<Base>(QObject *parent) : Base(parent)
     {
+      qRegisterMetaType<ObjectId>();
+      qRegisterMetaTypeStreamOperators<ObjectId>();
```

Comment:
Are those needed here? Looks like the ProbeControllerInterface does this already?